src/app/shared/ui/header/header.component.scss

```diff
@@ -0,0 +1,29 @@
+@media only screen and (max-width: 568px) {
+  .header__initials {
+    display: none;
+    margin-left: 15px;
+  }
+  .header__title {
+    margin-left: 20px;
+    font-size: var(--h2_font-size);
+  }
+}
+@media only screen and (min-width: 568px) {
+  .header__title {
+    margin-left: 20px;
+    font-size: var(--h1_font-size);
+  }
+  .header__initials {
+    margin-left: 15px;
+  }
+}
+.header__notifications-btn {
+  margin-right: -10px;
+  margin-left: -10px;
+}
+.header__example-spacer {
+  flex: 1 1 auto;
```

Comment:
chyba wystarczy tutaj flex: auto

"flex: auto - This is equivalent to flex: 1 1 auto" 
https://css-tricks.com/almanac/properties/f/flex/